docs/usage.md

```diff
@@ -132,6 +132,10 @@ If you prefer, you can use [HISAT2](https://ccb.jhu.edu/software/hisat2/index.sh
 
 To use HISAT2, use the parameter `--aligner hisat2` or set `params.aligner = 'hisat2'` in your config file.
 
+When STAR has been chosen as the aligner then the pipeline sets the minimal length of an alignment to 15. This is to avoid some strange very short mappings that can come from the soft-clipping in STAR. If you which to change this value then you can modify it with the following flag:
+
+`--params.min_aln_length `
```

Comment:
Nice! Still need to remove the `params` though..

Reply:
oh, I made that change. Looks like it didn't make it into to commit